core/base/document.php

```diff
@@ -1065,6 +1066,28 @@ public function delete() {
 		return $deleted && ! is_wp_error( $deleted );
 	}
 
+	/**
+	 *
+	 * @since 3.6.0
+	 *
+	 * @param array $elements
+	 *
+	 * @param array $map_old_new_post_ids
+	 */
+	public function on_import_replace_dynamic_content( &$elements, $map_old_new_post_ids ) {
+		foreach ( $elements as &$element_config ) {
+			$element_instance = Plugin::$instance->elements_manager->create_element_instance( $element_config );
+
+			if ( $element_instance ) {
+				$element_config = $element_instance->on_import_replace_dynamic_content( $element_config, $map_old_new_post_ids );
+
+				$this->on_import_replace_dynamic_content( $element_config['elements'], $map_old_new_post_ids );
+			}
+		}
+
+		$this->save_elements( $elements );
```

Comment:
Return the updated config, like the base method.
Save the elements from the caller `save_elements_of_imported_posts`